.python3-version

```diff
@@ -0,0 +1 @@
+3.9.5
```

Comment:
Could we use a more recent version? 3.9 end of life was in 2022

Reply:
Agreed that we should bump it ! I'll do it in another PR though !